src/sentry/models/integrations/external_actor.py

```diff
@@ -21,7 +21,9 @@
 class ExternalActor(DefaultFieldsModel):
     __include_in_export__ = False
 
-    actor = FlexibleForeignKey("sentry.Actor", db_index=True, on_delete=models.CASCADE)
+    # actor = FlexibleForeignKey("sentry.Actor", db_index=True, on_delete=models.CASCADE)
+    team = FlexibleForeignKey("sentry.Team", null=True, db_index=True, on_delete=models.CASCADE)
+    user_id = HybridCloudForeignKey("sentry.User", null=True, db_index=True, on_delete="CASCADE")
```

Comment:
I'm assuming you'll need to break these changes apart so that we can deploy schema, backfill, then change logic?